Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM python:2.7
 ENV PYTHONUNBUFFERED 1
 
+# build arguments
+ARG LUCIDA_HOSTNAME
+ENV LUCIDA_HOSTNAME $LUCIDA_HOSTNAME
```

Comment:
Why do you need these?

Reply:
I likely don't, I just copied these from the original business logic template server